core/src/main/scala/org/apache/spark/deploy/SparkSubmitArguments.scala

```diff
@@ -76,6 +75,7 @@ private[deploy] class SparkSubmitArguments(args: Seq[String], env: Map[String, S
   var proxyUser: String = null
   var principal: String = null
   var keytab: String = null
+  var dynamicAllocationEnabled: String = null
```

Comment:
Add `private` since this variable is never used out of this class.

Reply:
used in ut, so should i modify test or keep it `public`

Reply:
Yes, I think you can remove that variable check in UT, seems not so necessary.